src/GeositeFramework/js/Plugin.js

```diff
@@ -401,6 +405,10 @@ require(['use!Geosite',
             view.$uiContainer = $uiContainer;
 
             $uiContainer
+                // Call `hibernate` method on plugin when the eye icon is clicked
+                .find('.plugin-eye').on('click', function () {
+                    model.hibernate();
+                }).end()
```

Comment:
At least, this is the code path executed on the `develop` branch when the minimize button is pressed.